src/modules/interface/powertoy_module_interface.h

```diff
@@ -39,11 +39,11 @@ class PowertoyModuleIface
     /* Describes a hotkey which can trigger an action in the PowerToy */
     struct Hotkey
     {
-        bool win = false;

```

Comment:
let's not remove the inline initialization, since it's much safer to have all these fields always initialized.

Reply:
OK